README.md

```diff
@@ -39,6 +39,23 @@ And more:
 
 An introduction to the RGL API along with an example can be found [here](docs/Usage.md).
 
+## Building in Docker (Linux)
+
+1. Download [NVidia OptiX](https://developer.nvidia.com/designworks/optix/download) 7.2
+2. `export OptiX_INSTALL_DIR=<Path to OptiX>`
+3. `docker build . --tag rgl`
+4. Set up [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker):
+    - `distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
+      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
+      && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
+            sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
+            sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list`
+    - `sudo apt-get update`
+    - `sudo apt-get install -y nvidia-docker2`
+    - `sudo systemctl restart docker`
+4. `docker run --net=host --gpus all -v $(pwd):/code -v ${OptiX_INSTALL_DIR}:/optix -e OptiX_INSTALL_DIR=/optix -e NVIDIA_DRIVER_CAPABILITIES=all -it rgl /bin/bash`
+5. `./setup.bash --cmake --make -j`
```

Comment:
Considering the previous suggestion with two dockers, these scripts should contain several parameters (e.g., imageName, imageTag), which is becoming complicated. I think lines 48-55 could be deleted because the link under `NVIDIA Container Toolkit`contains these instructions. After that README will be compacted to a minimum.

Reply:
It's OK to remove nvidia-docker installation instructions, let users always use up-to-date instructions.
I also see the point with script parameters. We still need to specify it manually though.